src/views/SpaceSettings.vue

```diff
@@ -412,6 +413,16 @@ onMounted(() => {
                 <Icon name="github" />
               </template>
             </UiInput>
+            <UiInput
+              v-model="form.website"
```

Comment:
Validation doesn't seem to be working correctly, 

this is valid but shouldn't be
<img width="636" alt="image" src="https://user-images.githubusercontent.com/51686767/157172914-87139ea8-47d7-4592-b89d-75e5e6254f15.png">

Reply:
Also I think we should accept url without https://, like google.com